error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssertJMapRules.java

```diff
@@ -158,6 +158,11 @@ AbstractBooleanAssert<?> before(Map<K, V> map, K key) {
       return assertThat(map.containsKey(key)).isTrue();
     }
 
+    @BeforeTemplate
+    AbstractCollectionAssert<?, Collection<? extends K>, K, ?> before2(Map<K, V> map, K key) {
+      return assertThat(map.keySet()).contains(key);
+    }
```

Comment:
We generally prefer `Refaster.anyOf` in cases like these. (That does hide the different return types, so might be something we change our mind on later, but likely only if/when we have an Error Prone check to suggest one or the other format as appropriate.)